fsh/config.yaml

```diff
@@ -1,54 +0,0 @@
-# ╭─────────────────────────────ImplementationGuide-de.bbmri.fhir.json─────────────────────────────╮
```

Comment:
Please move (rename) the config instead of deleting and recreating it.

Reply:
Done.